components/filter/filter-dimension-set.js

```diff
@@ -45,7 +50,7 @@ class FilterDimensionSet extends LitElement {
 			 * Whether to render the selected items at the top of the filter
 			 * @type {boolean}
 			 */
-			selectedFirst: { type: Boolean, attribute: 'selected-first' },
+			selectedFirst: { type: Boolean, attribute: 'selected-first', reflect : true },
```

Comment:
Is the `reflect` needed to work, or is it just to tell the consumer we changed the value on them?